domains/hikamoru.json

```diff
@@ -4,6 +4,6 @@
         "email": "me.thefarkhodov@gmail.com"
     },
     "record": {
-        "CNAME": "hikamoru.uz"
+        "CNAME": "amoreforever.github.io/hikamoru/"
```

Comment:
`CNAME` can't contain `/`

Reply:
why hikamoru.uz isn't working? even hikamoru.uz site is available

Reply:
> why hikamoru.uz isn't working? even hikamoru.uz site is available

are you sure you verified the domain on the hosting provider of hikamoru.uz?

Reply:
> why hikamoru.uz isn't working? even hikamoru.uz site is available

the current domain shows `ERR_SSL_VERSION_OR_CIPHER_MISMATCH` so it's probably an issue with the SSL